src/scheduler/worker.rs

```diff
@@ -50,7 +50,7 @@ pub struct Worker<C: Context> {
     pub parked: AtomicBool,
     scheduler: Arc<Scheduler<C>>,
     local: WorkerLocalPtr,
-    pub local_works: WorkBucket<C>,
+    pub local_work: WorkBucket<C>,
```

Comment:
I suggest considering a rename to `local_bucket` or `local_work_bucket`.

Reply:
Done.